### PR TITLE
Delete, create, edit FAQ for admin

### PR DESCRIPTION
**Pull request by MichelleCCodes**
## Description
- FAQ flow: edit, add, delete
## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have checked my code and corrected any misspellings
- [x] Size of pull request kept to a minimum

### CODE REVIEW

**Review by JonathanMary — APPROVED** [review]
All changes good, needs some linting.

File: api/faq/faqModel.js [hunk 1]
```diff
@@ -14,11 +14,11 @@ const create = async (faq) => {
 };
 
 const update = (id, faq) => {
```
Comment (dgamboa) [inline api/faq/faqModel.js:16]: We could refactor the id param here to faq_id to match the rest of the code, right?

File: api/faq/faqRouter.js [hunk 2]
```diff
@@ -24,14 +24,14 @@ router.get('/', authRequired, function (req, res) {
...
-  const id = String(req.params.id);
+router.get('/:faq_id', authRequired, function (req, res) {
+  const id = req.params.faq_id;
   FAQ.findById(id)
     .then((faq) => {
       if (faq) {
         res.status(200).json(faq);
       } else {
-        res.status(404).json({ error: 'faqNotFound' });
+        res.status(404).json({ error: 'faq NotFound' });
```
Comment (dgamboa) [inline api/faq/faqRouter.js:34]: How about a space between "Not" and "Found" and maybe lower case?

File: api/faq/faqRouter.js [hunk 3]
```diff
@@ -69,18 +69,13 @@ router.put('/:id', authRequired, (req, res) => {
 
 router.delete('/:id', authRequired, (req, res) => {
```
Comment (dgamboa) [inline api/faq/faqRouter.js:70]: Should we change id to faq_id for the path here to match the other endpoints?

**Review by dgamboa — APPROVED** [review]
Looks good, just a couple of minor suggestions that can be implemented in another PR.

File: api/faq/faqModel.js (hunk 1 above)
Comment (MichelleCCodes) [reply api/faq/faqModel.js:16]: Good catch! That would accurately reflect the changes made elsewhere.

File: api/faq/faqRouter.js (hunk 3 above)
Comment (MichelleCCodes) [reply api/faq/faqRouter.js:70]: Yes, I agree! It would be great maintenance of the codebase to keep it all consistent!

**Pull request merged**